### PR TITLE
Fix broken link to location descriptor reference

### PR DESCRIPTION
**Pull request by viniciusdacal**
the current link is broken.

### CODE REVIEW

**Comment by taion** [comment]
Oof. Can you fix the "more details" link in the glossary to point to the v2.x branch on `history` as well?

**Comment by viniciusdacal** [comment]
@taion for sure :)

File: docs/Glossary.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ A *hash* is a string that represents the hash portion of the URL. It is synonymo
 type LeaveHook = (prevState: RouterState) => any;
 ```
 
-A *leave hook* is a user-defined function that is called when a route is about to be unmounted. It receives the previous [router state](#routerstate) as its first argument. 
+A *leave hook* is a user-defined function that is called when a route is about to be unmounted. It receives the previous [router state](#routerstate) as its first argument.
```
Comment (viniciusdacal) [inline docs/Glossary.md]: oh, there is a diff here, it was just the 'trailing spaces'. If there is a problem, I can revert.

**Comment by viniciusdacal** [comment]
updated 😄

**Comment by taion** [comment]
Thanks!

**Pull request merged**